ladybug/designday.py

```diff
@@ -1369,12 +1464,22 @@ def to_dict(self):
 class RevisedClearSkyCondition(SkyCondition):
     """An object representing an ASHRAE Revised Clear Sky (Tau model).
 
-    Properties:
+    Args:
         month
         day_of_month
         tau_b
         tau_d
         daylight_savings_indicator
+
```

Comment:
I see I forgot a docstring here. Can you add the following:
```
        month: Month in which the design day occurs.
        day_of_month: Day of the month on which the design day occurs.
        tau_b: Value for the 'beam' term in the Tau model. Typically
            found in .stat files.
        tau_d: Value for the 'diffuse' term in the Tau model. Typically
            found in .stat files.
        daylight_savings_indicator: Text ('Yes' or 'No'), for whether daylight savings
            time is active. Default: 'No'
```

Reply:
Done!